.circleci/config.yml.in

```diff
@@ -192,8 +192,9 @@ jobs:
       - checkout
       - run:
           command: |
-            pip install --user --progress-bar off flake8 typing
-            flake8 --config=setup.cfg .
+            pip install --user --progress-bar off pre-commit
+            pre-commit install-hooks
```

Comment:
I wonder if we can delete this line?
I think `run --all-files` should work even without having the hooks? At least that's what our instructions in the README suggest.

CI failures are unrelated BTW, it's good to merge on my side. I'll let you do it in case there's any leftovers

Reply:
I like to keep the actual check separate from the installation. This way, the last step in the workflow only contains the information that a contributor needs in case the workflow fails. Granted, the installation of the hook environments does not create a lot of noise, but still.

Reply:
@NicolasHug 
FYI.
https://github.com/pytorch/pytorch/issues/64906